src/main/java/org/broadinstitute/hellbender/utils/test/BaseTest.java

```diff
@@ -168,25 +90,6 @@ protected static String getNonNullEnvironmentVariable(String envVarName) {
         return value;
     }
 
-    @BeforeClass
-    public void initGenomeLocParser() throws FileNotFoundException {
-        hg19ReferenceReader = new CachingIndexedFastaSequenceFile(new File(hg19MiniReference));
-        hg19Header = new SAMFileHeader();
-        hg19Header.setSequenceDictionary(hg19ReferenceReader.getSequenceDictionary());
-        hg19GenomeLocParser = new GenomeLocParser(hg19ReferenceReader);
-    }
-
-    protected List<GenomeLoc> intervalStringsToGenomeLocs( String... intervals) {
-        return intervalStringsToGenomeLocs(Arrays.asList(intervals));
-    }
-
-    protected List<GenomeLoc> intervalStringsToGenomeLocs( List<String> intervals ) {
-        List<GenomeLoc> locs = new ArrayList<>();
-        for (String interval: intervals)
-            locs.add(hg19GenomeLocParser.parseGenomeLoc(interval));
-        return Collections.unmodifiableList(locs);
-    }
-
```

Comment:
For purity sake I think getToolTestDataDir() should be left abstract in BaseTest and instantiated in GATKBaseTest.

Reply:
This can be overridden, but I think that it is a good practice to have a standard folder structure for test data directories. In addition, this is not project specific, because the java source structure (`src/test/resources`) is more or less an standard.

No action for this comment.